test/MUXControlsTestApp/InfoBarTestPage.xaml.cs

```diff
@@ -0,0 +1,214 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using MUXControlsTestApp.Shared;
+using Windows.UI.Popups;
+
+
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace MUXControlsTestApp.Shared
+{
+    [TopLevelTestPage(Name = "InfoBar")]
+    public sealed partial class InfoBarTestPage : Page
+    {
+
+
+        InfoBarSeverity severity;
```

Comment:
Add a way to cancel the closing event so you can test that functionality. Something like a check box that the test app checks and cancels the closing event in the checkbox is checked.

Reply:
Good idea! I created a checkbox on the TestPage for cancel in the way you described.